Description: Fixes $mathend_mark
 This patch fixes math.pl to default to empty $mathend_mark to prevent
 mathend000# in documents.
Author: Arwin Vosselman <arwin@xs4all.nl>
Bug-Debian: http://bugs.debian.org/424897

--- latex2html-2008-debian1.orig/versions/math.pl
+++ latex2html-2008-debian1/versions/math.pl
@@ -39,7 +39,7 @@ if ($HTML_OPTIONS =~ /math/) {
 
 package main;
 
-$mathend_mark = "\n${verbatim_mark}mathend000#";
+$mathend_mark = "";
 $keepcomments_rx = "\\s*(picture|makeimage|xy|diagram|mathend)[*]?"
   unless ($keepcomments_rx =~ /mathend/);
 

